Assets/HoloToolkit/Input/Scripts/Interactions/TapToPlace.cs

```diff
@@ -190,7 +195,7 @@ private void ToggleSpatialMesh()
             }
         }
 
-        private Bounds CalculateAllColliderBounds()
+        private void CalculateColliderOffset()
```

Comment:
It would be nice to make this public, and update all the offset positional info inside.